packages/wrangler/scripts/bundle.ts

```diff
@@ -42,9 +55,29 @@ async function run() {
     define: {
       "process.env.NODE_ENV": '"production"',
     },
+    watch: flags.watch ? watchLogger("Miniflare") : false,
   });
 }
 
+async function run() {
+  // main cli
+  await buildMain();
+
+  // custom miniflare cli
+  await buildMiniflareCLI();
+
+  console.log("Built wrangler & Miniflare CLI.");
```

Comment:
```suggestion
```

We shouldn't be saying we built the miniflare cli (we don't mention miniflare anywhere either), but let's not add this anyway?